### PR TITLE
Marsifi/onboarding week1

### PR DESCRIPTION
**Pull request by marsifi**
This PR contains a few minor changes to the clp-tutorial which had some outdated commands..

### CODE REVIEW

File: x/clp/keeper/msg_server.go [hunk 1]
```diff
@@ -349,9 +350,10 @@ func (k msgServer) RemoveLiquidity(goCtx context.Context, msg *types.MsgRemoveLi
 
 func (k msgServer) CreatePool(goCtx context.Context, msg *types.MsgCreatePool) (*types.MsgCreatePoolResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
+
 	// Verify min threshold
-	MinThreshold := sdk.NewUintFromString(types.PoolThrehold)
-	if msg.NativeAssetAmount.LT(MinThreshold) { // Need to verify
```
Comment (marsifi) [inline x/clp/keeper/msg_server.go:354]: This is where the hardcoded PoolThreshold was being used

**Comment by timlind** [comment]
Thanks @marsifi. Can you put the param change into a it's own PR so that we can handle them separately and get this one merged in?

**Comment by marsifi** [comment]
@timlind I removed the param changes

File: x/clp/keeper/msg_server.go (hunk 1 above)
Comment (Jedi2002) [reply x/clp/keeper/msg_server.go]: hi @marsifi  , This number was decided by the product team as the min amount of rowan required to create a pool . At that time we decided not to apply an limits on the external asset

**Review by Jedi2002 — APPROVED** [review]

**Pull request merged**